torch/tensor.py

```diff
@@ -418,10 +418,7 @@ def __rpow__(self, other):
 
     @_wrap_type_error_to_not_implemented
     def __floordiv__(self, other):
-        result = self / other
-        if result.dtype.is_floating_point:
-            result = result.trunc()
-        return result
+        return torch.floor_divide(self, other)
 
     @_wrap_type_error_to_not_implemented
     def __rfloordiv__(self, other):
```

Comment:
In order to support `rfloordiv` I would have add another schema for `floor_div(Scalar input, Tensor other)`. I don't  know if this is really worth adding to native registration. We don't have `div(Scalar input, Tensor other)` in native functions, and `rfloordiv` is really a python-only concept (`a // b`, if floordiv isn't defined on a then call rfloordiv on b).